charts/vh-admin-web/Chart.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v2
 name: vh-admin-web
 home: https://github.com/hmcts/vh-admin-web
-version: 0.0.3
+version: 0.0.1
```

Comment:
Should this be a lower version than before?

Reply:
Yea, was just testing the helm chart pushing. Terrible news: it's not working.